extensions/test-matchers/src/matchers/errors/toBeRevertedWithError.spec.ts

```diff
@@ -0,0 +1,374 @@
+import { tevmDefault } from '@tevm/common'
```

Comment:
in future prs if possible try using the test matchers in actual tevm tevm tests. Good job for AI actually to look for good use cases to use a specific test util

Reply:
Only needs to be used in 1 btw but it's a great way to get extra confidence while also improving the existing tests trying to get context in there so ai starts using them too